core/includes/diff.inc

```diff
@@ -774,9 +774,8 @@ class MappedDiff extends Diff {
    *
    * Computes diff between sequences of strings.
    *
-   * This can be used to compute things like
-   * case-insensitve diffs, or diffs which ignore
-   * changes in white-space.
+   * This can be used to compute things like case-insensitive diffs, or diffs
+   * which ignore changes in white-space.
```

Comment:
Is the dash in `white-space` really necessary here?

Reply:
Maybe not, but considering it's used a few times throughout core, I'd suggest that might belong to a different issue (it's not a spelling mistake).

Reply:
Grammatically, one uses a hyphen when the two words are used as an adjective to modify another, but no hyphen if the adjective is modifying a noun. Thus:

* Ignore changes in white space.
* That was a white-space change.